compiler/tools/fpp-to-cpp/test/struct/alias_type.fpp

```diff
@@ -1,4 +1,9 @@
 type U16Alias = U16
 type T
 type TAlias = T
-struct AliasType { x: U16Alias, y: TAlias }
+type SAlias = string size 50
+struct AliasType {
+    x: U16Alias,
+    y: TAlias,
+    z: [50] SAlias
```

Comment:
Maybe we can reduce this array size from 50 to 10? To cut down on the amount of generated test code.